hr_planning_resources/models/hr_task.py

```diff
@@ -0,0 +1,519 @@
+from datetime import datetime, time
+
+import pytz
+from dateutil.relativedelta import relativedelta
+
+from odoo import _, api, fields, models
+
+TASK_TYPES = [
+    ("task", _("Task")),
```

Comment:
```suggestion
    ("task", "Task"),
```
You don't need to mark for translation selection labels